frontend/build.sbt

```diff
@@ -111,4 +111,6 @@ libraryDependencies ++= frontendDependencies.map {
      .exclude("org.slf4j", "slf4j-simple") // snatches SLF4J logging from Logback, our chosen logging system
 }
 
+dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % "2.11.4"
```

Comment:
this is needed to prevent it using 2.12 which causes play's akka to fall over at runtime objecting that it needs 2.11.x